diff --git a/dist/clients/nuxt/utils.ts b/dist/clients/nuxt/utils.ts
index cf818c000967a83a8a2d25f6c5f636e5d04a1d7c..0eae687a79de0d7143962fa7fcbb67df5f0090aa 100644
--- a/dist/clients/nuxt/utils.ts
+++ b/dist/clients/nuxt/utils.ts
@@ -355,6 +355,10 @@ export const unwrapRefs = <T>(value: T): UnwrapRefs<T> => {
   if (value === null || typeof value !== 'object' || value instanceof Headers) {
     return (isRef(value) ? unref(value) : value) as UnwrapRefs<T>;
   }
+  
+  if (value instanceof Blob) {
+    return value as UnwrapRefs<T>;
+  }
 
   if (Array.isArray(value)) {
     return value.map((item) => unwrapRefs(item)) as UnwrapRefs<T>;
